tsdb/meta.go

```diff
@@ -1329,11 +1309,16 @@ func NewSeries(key string, tags map[string]string) *Series {
 }
```

Comment:
I'm going to leave this as a `sync.RWMutex` because I've had to change almost all of our uses of `sync.Mutex` to `sync.RWMutex` due to lock contention issues that appear under different workloads.